run.sh

```diff
@@ -58,6 +61,16 @@ while getopts ":ohn:" option; do
           exit
         fi
         ;;
+      ci) # Run Cypress tests in headless mode for CI
+        test_file=$OPTARG
+        path=$(pwd)
+        if [[ -ci $test_file ]]; then
```

Comment:
The last pipeline run in https://github.com/CirclesUBI/o-platform/actions/runs/5387980941/jobs/9780030955#step:16:75 also showed a regression with this line:

    ./run.sh: line 67: conditional binary operator expected

Is `-ci` an actual conditional accepted by bash? What is being tested here?